jmx/plugins/org.jboss.tools.jmx.jolokia/src/org/jboss/tools/jmx/jolokia/internal/connection/JolokiaMBeanServerConnection.java

```diff
@@ -293,15 +336,15 @@ public void unregisterMBean(ObjectName name)
 	
 	@Override
 	public ObjectInstance createMBean(String className, ObjectName name)
-			throws ReflectionException, InstanceAlreadyExistsException, MBeanRegistrationException, MBeanException,
+			throws ReflectionException, InstanceAlreadyExistsException, MBeanException,
```

Comment:
Any reason you delete only one of the exceptions? As far as I can tell, we're not throwing any of them. Why only remove MBeanRegistrationException?  Just curious